bank-of-anthos/extras/tls-ip-selfsigned/README.md

```diff
@@ -0,0 +1,37 @@
+# TLS Frontend with Self-Signed Certificates 
+
+This directory contains manifests for deploying the Bank of Anthos frontend with an HTTPS-only ingress, using self-signed certificates and a global static IP. 
+
+## Prerequisites 
+
+- A GCP project
+- 1 GKE cluster 
```

Comment:
`A GKE cluster`? "A project" vs "1 cluster" just reads clunky

Reply:
Addressed on the main bank-of-anthos repo: [Minor readability changes](https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/502)